### PR TITLE
test: move /e2e/ts_library ts_library tests to /packages/typescript/test

### PR DESCRIPTION
**Pull request by gregmagolan**
No functional changes to tests. Just moving them to the root workspace so they are tested with `bazel test ...`

### CODE REVIEW

File: packages/protractor/test/protractor-2/BUILD.bazel [hunk 1]
```diff
@@ -41,7 +34,7 @@ ts_library(
     name = "ts_spec",
     testonly = True,
     srcs = ["test.spec.ts"],
-    tsconfig = ":tsconfig-test",
+    tsconfig = "//packages/typescript/test:tsconfig-test",
```
Comment (alexeagle) [inline packages/protractor/test/protractor-2/BUILD.bazel]: this seems undesirable dependency between packages, why not have a copy of this tsconfig here if it's needed in protractor?

File: packages/typescript/test/foobar/BUILD.bazel [hunk 2]
```diff
@@ -1,6 +1,6 @@
-load("@npm_bazel_typescript//:index.bzl", "ts_library")
+load("@npm_bazel_typescript//:index.from_src.bzl", "ts_library")
 
-package(default_visibility = ["//visibility:public"])
+package(default_visibility = [])
```
Comment (alexeagle) [inline packages/typescript/test/foobar/BUILD.bazel]: isn't this the same as no `package` declaration?

File: WORKSPACE [hunk 3]
```diff
@@ -110,6 +110,16 @@ node_repositories(
 
 yarn_install(
     name = "npm",
+    manual_build_file_contents = """
+filegroup(
```
Comment (alexeagle) [inline WORKSPACE]: this is really surprising; add a comment saying why this is needed?

File: packages/typescript/test/foobar/BUILD.bazel (hunk 2 above)
Comment (gregmagolan) [reply packages/typescript/test/foobar/BUILD.bazel]: 👍 not sure how that go in there. thanks

File: packages/protractor/test/protractor-2/BUILD.bazel (hunk 1 above)
Comment (gregmagolan) [reply packages/protractor/test/protractor-2/BUILD.bazel]: 👍

File: WORKSPACE (hunk 3 above)
Comment (gregmagolan) [reply WORKSPACE]: 👍

**Pull request merged**